manifest_staging/helm/gatekeeper/templates/config-customresourcedefinition.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.4
+    helm.sh/hook: crd-install
+    helm.sh/hook-delete-policy: before-hook-creation
+  creationTimestamp: null
+  labels:
+    app: '{{ template "gatekeeper.name" . }}'
+    chart: '{{ template "gatekeeper.name" . }}'
+    gatekeeper.sh/system: "yes"
+    heritage: '{{ .Release.Service }}'
+    release: '{{ .Release.Name }}'
```

Comment:
`-useCRDsDir` with Helm 3 doesn't seem to like the templated variables for labels in CRDs

Reply:
Ack, if we use helm 3, we would need to re-work how CRDs are kustomized slightly